samples/highcharts/website/static-editor/demo.details

```diff
@@ -0,0 +1,7 @@
+---
+ name: Highcharts Icon for Highcharts Editor
+ authors:
+   - Nancy Dillon
+ js_wrap: b
+ skipTest: true
```

Comment:
When `skiptTest: true`, these charts are not considered when doing visual regression tests. But if there's not a specific reason for leaving them out (like custom animation, timeouts etc.), all the static charts should be part of the regression tests.